src/components/Button/SubmitButton.js

```diff
@@ -71,7 +71,7 @@ class SubmitButton extends React.Component {
       return <span>{pristineContents}</span>;
     }
     if (loading) {
-      return <Loader size="14px" />;
```

Comment:
On my display, setting Loader size to 16px does not keep the height constant between the states (Pristine, Loading, etc...).  Also, do we want the Ring size to increase, or do we want the overall button height to be constant with contents centered vertically? You might want to check w/ other Forge team members about the best way to fix it.

Reply:
@tedwards-bw The requirements should come from UX.

Reply:
changed the loader size back and adjusted buttons min-height

Reply:
Should the min-height be confined to Button.Submit (the only thing that replaces its contents with a Loader)?  Would Button.Small, for example, be impacted by this?

Reply:
I changed the solution to affect only submit button